app/styles/libs/_helpers.scss

```diff
@@ -98,3 +98,10 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 .no-wrap {
   white-space: nowrap;
 }
+
+.extra-link {
```

Comment:
there is already a class defined name ```truncate``` which add '...' if text is long. Pls use this